### PR TITLE
Updated repo add and job wait command in notes

### PR DESCRIPTION
**Pull request by tusharraut1994**
Signed-off-by: Tushar Raut tushar@portworx.com

- Updated chart repo add command in README.md
- Updated job wait command in NOTES.txt

### CODE REVIEW

File: charts/px-backup/templates/NOTES.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ Monitor PX-Backup Install:
 --------------------------------------------------
 Wait for px-backup status to be in "Completed" state.
 
-    kubectl get po --namespace {{ .Release.Namespace }} -ljob-name=pxcentral-post-install-hook  -o wide | awk '{print $1, $3}' | grep -iv error
+    kubectl --namespace {{ .Release.Namespace }} wait --for=condition=complete --timeout=120s job/pxcentral-post-install-hook
```
Comment (disrani-px) [inline charts/px-backup/templates/NOTES.txt:9]: Increase the timeout to 5 mins, pulling images can take much longer than 2 mins

Comment (tusharraut1994) [reply charts/px-backup/templates/NOTES.txt:9]: changed.

File: charts/px-backup/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ To install the chart with the release name `px-backup`:
 
 Add portworx/px-backup helm repository using command:
 ```console
-$ helm repo add portworx https://raw.githubusercontent.com/portworx/helm/master/stable/
+$ helm repo add portworx http://charts.portworx.io/
```
Comment (sharma-tapas) [inline charts/px-backup/README.md:11]: is this endpoint up, did we test it?

Comment (tusharraut1994) [reply charts/px-backup/README.md:11]: yes, verified the endpoint.

**Pull request merged**